lsp-metals.el

```diff
@@ -231,6 +231,15 @@ change it again."
   (interactive)
   (lsp-send-execute-command "reset-choice" ()))
 
+(defun lsp-metals-copy-worksheet-output ()
+  "Copy worksheet with evaluated results as comments."
+  (interactive)
+  (let ((command-result (lsp-send-execute-command "metals.copy-worksheet-output" (lsp--buffer-uri))))
+    (if (lsp-get command-result :value)
```

Comment:
nit: if -> when and remove and you can remove the progn. After this it looks good to me from elisp perspective, and I will let @kurnevsky to test the metals part and merge it.

Reply:
yup, that looks cleaner.